### DISCUSSION TITLE
normal vs fast guides

### DISCUSSION THREAD

**Discussion by twospirited** [xanthics/gw2craft#61] [Ideas]
any way to have the normal not list any tp purchases. save the tp stuff for the fast guide only? i think it'd be easier, esp for those not used to the page setup, and being overwhelmed by it. as i've heard a few in guidls i'm in say it's a bit much to look at in one go right now.

**Comment by xanthics** [comment]
f2p guides already don't include any partially crafted(Mixed) items, as they cannot purchase them.  As a side effect of that, everything in the purchase list can also be found in the game world EG https://gw2crafts.net/f2p/artificing.html

As such I believe this request is already solved?

**Comment by twospirited** [comment]
when i click that and put in my API key, it gives me a Mixed (Buy on TP) section. all i'm asking is why not remove this so it's all just straight up gathering or karma purchases as needed

**Comment by xanthics** [comment]
I'm not seeing the behavior you are describing.  There is no Mixed section after entering my API key.  if you were on a page with f2p in the url please share it.  

![image](https://user-images.githubusercontent.com/3137623/196065919-a79cd8ab-db2c-4919-85ad-25b9023f5d41.png)

![image](https://user-images.githubusercontent.com/3137623/196067052-b7359b27-0f9c-4cfb-8a7c-b3a56042bc99.png)

**Comment by twospirited** [comment]
![image](https://user-images.githubusercontent.com/45153697/196067446-fed47448-cbf4-48be-95e1-e6ea0b21bf2a.png)
is what mine shows just after COLLECTIBLES(Check Bank First or Buy on TP) and making when i check mine there
and for Jewelcrafting, 
![image](https://user-images.githubusercontent.com/45153697/196067526-3dfbd850-af10-4467-8fb9-9a3ca995521f.png)

**Comment by xanthics** [comment]
You aren't on a free to play guide.  You are at https://gw2crafts.net/jewelcraft.html not https://gw2crafts.net/f2p/jewelcraft.html

**Reply by xanthics** [reply]
Same with artificing.  https://gw2crafts.net/artificing.html vs https://gw2crafts.net/f2p/artificing.html

**Comment by twospirited** [comment]
most don't see the f2p, they, like me, just go normal vs fast guide why not merge f2p with the normal then?
